keyboards/kbdfans/kbd67mkiirgb/config.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include "config_common.h"
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1224
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    DZTECH
+#define PRODUCT         kbd67mkiirgb
+#define DESCRIPTION     kbd67mkii rgb keyboard
+
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 15
+#define MATRIX_ROW_PINS { B1, B10, B11, B14, B12 }
+#define MATRIX_COL_PINS {A6, A7, B0, B13, B15, A8, A15, B3, B4, B5, B8, B9, C13, C14, C15 }
+#define DIODE_DIRECTION COL2ROW
+
+#define RGB_MATRIX_LED_PROCESS_LIMIT 4
+#define RGB_MATRIX_LED_FLUSH_LIMIT 26
+#define DEBOUNCE 3
```

Comment:
Debounce is pretty low.  This may cause issues, depending on the switches and other issues. 

However, if you're fine with this, then that's fine!